packages/formstr-app/public/index.html

```diff
@@ -25,6 +25,12 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Formstr</title>
+    <script>
+      window.__FORMSTR__FORM_IDENTIFIER__ = {
+        naddr: '@naddr',
+        viewKey: '@viewKey'
```

Comment:
is this a required param?

Reply:
not really. depends on the form. if its public, then it should be useless, else it will be populated. Idea is that the form creator should be able to share this form html manually. The html should be self contained.